packages/edit-site/src/components/layout/index.js

```diff
@@ -154,7 +138,7 @@ export default function Layout() {
 	// Sets the right context for the command palette
 	let commandContext = 'site-editor';
 
-	if ( canvasMode === 'edit' && isEditorPage ) {
```

Comment:
I looked around and haven't found any page with `canvasMode=edit` that wasn't in the editor.

Reply:
Also interacted with the command palette in various places, and it seems to work as expected.